src/screens/TopicMemberScreen/index.js

```diff
@@ -154,7 +154,6 @@ const TopicMemberScreen = () => {
     cancelTokenRef.current = axios.CancelToken.source();
   };
 
-  if (isInitialLoading) return null;
   return (
     <SafeAreaProvider forceInset={{top: 'always'}} style={styles.parentContainer}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The check for `isInitialLoading` has been removed. If this was intentional and loading state is handled elsewhere, then it's fine. However, if not, this could lead to rendering issues or errors while data is still being fetched.
```diff
+  if (isInitialLoading) return null;
   return (
     <SafeAreaProvider forceInset={{top: 'always'}} style={styles.parentContainer}>
       <NavHeader
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->